en/docs/learn/consume-api/invoke-apis/invoke-apis-using-tools/try-out-using-postman.md

```diff
@@ -0,0 +1,57 @@
+# Try out using Postman.
+
```

Comment:
The filename "Screenshot from 2020-10-08 12-43-56.png" that you have added separately needs to be renamed as this is not a proper file name. In addition, it does not look like the image is referred to in this document.